### PR TITLE
Moved org/impl specific vars out for playbook scripts. They should be…

### PR DESCRIPTION
**Pull request by ankanm**
… in group_vars/host_vars alongwith inventory files.

~~**WIP**~~

### CODE REVIEW

**Comment by amitbose327** [comment]
Looks good so far, but waiting for other Moz related stuff to move out from seomoz/roger-monitoring.

**Comment by ankanm** [comment]
Done.

File: ansible/roles/monitoring-influxdb9/tasks/main.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     - monitoring
 
 - name: Download influxdb package
-  get_url: url="http://registry.roger.dal.moz.com:81/roger/pkgs/influxdb_{{ influxdb_version }}_amd64.deb" dest="{{ monitoring_tmp_dir }}"
+  get_url: url="{{ influxdb_download_url_path }}influxdb_{{ influxdb_version }}_amd64.deb" dest="{{ monitoring_tmp_dir }}"
```
Comment (amitbose327) [inline ansible/roles/monitoring-influxdb9/tasks/main.yml]: This should be "influxdb_download_url_path_prefix"

**Comment by amitbose327** [comment]
LGTM, apart from the above comment.

**Pull request merged**